packages/eslint-config-core-js/index.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
 		'no-underscore-dangle': ['error', { allowAfterThis: true }],
 		'no-void': 'error',
 		semi: 'error',
+		'max-len': ['error', { code: 140, ignorePattern: '^import .*' }],
```

Comment:
140 can be changed of course.

Reply:
Why would we need to add this if we use prettier?

Reply:
We have prettier rule in handbook https://infinum.com/handbook/frontend/react/getting-started/ecosystem#prettier and line lenght is fixated to 120. I think we should align linter with prettier.

Reply:
It was a long time ago, but someone requested to have a linter ignore the warning and/or error for the long imports that prettier couldn't restructure.
I will put 120 then.

Reply:
Done. It is now 120.